Models/user.py

```diff
@@ -12,6 +12,7 @@ class User(Base):
     password: Mapped[str] = mapped_column()
     id_collection: Mapped[int] = mapped_column(ForeignKey("Collection.id"))
     city: Mapped[str] = mapped_column()
+    warderobe: Mapped[str] = mapped_column()
```

Comment:
Powinno być Mapped[Optional[str]] - możemy dodać usera bez szafy. Inaczej za każdym razem trzeba dodać usera z tym stringiem
Opcjonalnie druga ścieżka to zmodyfikowaćwszystie endpointy i testy żeby dodawały usera z szafą